### PR TITLE
alpine, mount management, path removal

### PR DESCRIPTION
**Pull request by otakup0pe**
cc @ericfode

### CODE REVIEW

File: aomi/seed.py [hunk 1]
```diff
@@ -483,14 +526,35 @@ def approle(client, approle_obj, opt):
     client.create_role(name, **role_obj)
 
 
+def generated_key(key, opt):
```
Comment (ericfode) [inline aomi/seed.py]: Maybe have this take a map of generators. Or have it introspect and search for any classes that are loaded that implement `gen_type` and generate.
Then you could have a n8ce littpe decorator to extend it with like
```
Import aomi.generators

@generator("super_random")
 Def random_gen (random_com_api_key: ''):
    Shutil.exec ("curl -H 'api_key: " + random_com_api_key + "' http://random.com")

File: aomi/seed.py [hunk 2]
```diff
@@ -500,25 +564,96 @@ def generated(client, obj, opt):
...
+            update_msg = "Writing %s generated secrets to %s" % \
+                         (genseclen, vault_path)
+            log(update_msg, opt)
+            client.write(vault_path, **secret_obj)
+    else:
+        log("Removing generated secret at %s" % vault_path, opt)
+        client.delete(vault_path)
+
+
+def mount_path(client, obj, opt):
```
Comment (ericfode) [inline aomi/seed.py]: This can fail. maybe_mount_path?

File: aomi/seed.py [hunk 3]
```diff
@@ -500,25 +564,96 @@ def generated(client, obj, opt):
...
+
+    backends = client.list_secret_backends()
+    mounted = is_mounted(path, backends, 'generic')
+    if obj.get('state', 'present') == 'present':
+        if not mounted:
+            actually_mount(client, 'generic', path)
+            log("Mounted %s" % (path), opt)
+    else:
+        if mounted:
+            unmount(client, 'generic', path)
```
Comment (ericfode) [inline aomi/seed.py]: Wat?

File: aomi/seed.py [hunk 4]
```diff
@@ -500,25 +564,96 @@ def generated(client, obj, opt):
...
+            log("Mounted %s" % (path), opt)
+
+
+def duo_enable(client, backend, opt):
```
Comment (ericfode) [inline aomi/seed.py]: Maybe break each of the mount types into their own namespaces. Also have a common namespace with functions like `ensure(verify_fn, action_fn, fail_fn)`

**Comment by otakup0pe** [comment]
Created https://github.com/Autodesk/aomi/issues/67 to track fixing up docstring, it's outside the scope of this PR.

File: aomi/seed.py (hunk 1 above)
Comment (otakup0pe) [inline aomi/seed.py]: We aren't going to be including bits of `aomi` in third party applications to generate secrets... Which is what this comment seems to imply?

File: aomi/seed.py (hunk 3 above)
Comment (otakup0pe) [inline aomi/seed.py]: This function handles mounting (or unmounting) paths. As in this function handles "mount paths" as opposed to this function "mounts a path".

File: aomi/seed.py (hunk 4 above)
Comment (otakup0pe) [inline aomi/seed.py]: I like this, but considering it out of scope for this PR.  https://github.com/Autodesk/aomi/issues/68 to track this work.

**Pull request merged**